sdk/servicebus/service-bus/review/service-bus.api.md

```diff
@@ -18,18 +17,6 @@ import { TokenType } from '@azure/core-amqp';
 import { WebSocketImpl } from 'rhea-promise';
 import { WebSocketOptions } from '@azure/core-amqp';
 
-// @public
-export type AuthorizationRule = {
-    claimType: string;
-    claimValue: string;
-    rights: {
-        accessRights?: string[];
-    };
-    keyName: string;
-    primaryKey?: string;
-    secondaryKey?: string;
-};
-
```

Comment:
Anything we can do to not make the diffs related to the removal of management apis appear in this PR?

Reply:
I guess this must have happened recently enough so probably not - this is literally the MD file form master at this point.